shex/sio-model/patientStatusShape.shex

```diff
@@ -0,0 +1,52 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v020/shex/>
+PREFIX obo: <http://purl.obolibrary.org/obo/>
+PREFIX sio: <http://semanticscience.org/resource/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX snomedct: <http://purl.bioontology.org/ontology/SNOMEDCT/>
+PREFIX hl7: <http://purl.bioontology.org/ontology/HL7/>
+
+
+:personShape IRI {
+  a [obo:NCBITaxon_9606];
+  sio:SIO_000228 @:personRoleShape;
+  (sio:SIO_000217 @:statusQualityShape |
+   sio:SIO_000217 @:statusQualityShape; 
+   sio:SIO_000217 @:deadDateQualityShape)
+}
+
+:personRoleShape IRI {
+  a [obo:OBI_0000093];
+  (sio:SIO_000356 @:statusProcessShape | 
+   sio:SIO_000356 @:statusProcessShape; 
+   sio:SIO_000356 @:deadDateProcessShape) 
+} 
+
+:statusProcessShape IRI {
+  sio:SIO_000229 @:outputShape
+}
+
+:outputShape IRI {
+  a [obo:PATO_0001422 obo:PATO_0001421  snomedct:399307001 hl7:C4291647]
```

Comment:
Question about the diagram: 
why the type `sio:realizable-entity` is not attached to the status node MV1? 
I see both **_gender_** and _**birth date**_ in the _**Personal Information**_ Module are attached.

Reply:
I think @markwilkinson might left this out in the diagram to improve the readability of the diagram. But I also wonder if we can make `owl file` and add all MV `rdf:type` as children of  `sio:relizable-entity`. What do you think about this solution @markwilkinson ?

Reply:
That sounds good to me.